app/templates/summary.html

```diff
@@ -0,0 +1,82 @@
+{% extends theme('layouts/_survey.html') -%}
+
+{% block page_title -%}Summary{% endblock -%}
+
+{% set not_answered = 'Not answered' %}
+
+{% block survey -%}
+<div class="header u-ta-l">
+  <h1 class="header__title saturn">Your responses</h1>
+  <h2 class="header__subtitle mars">Please check carefully before submission.</h2>
+</div>
+
+<div>
+  <dl class="summary">
+  {% for block in content %}
+
+    {% set summary_id = "summary-" ~ loop.index0 %}
+
+    <dt class="summary__title saturn" id="{{summary_id}}">{{block.title | safe}}</dt>
+    <dd class="summary__block">
+      <dl class="summary__items">
+
+      {% for item in block.items %}
+
+        {% set id = summary_id ~ "-" ~ loop.index0 %}
+
+        <dt class="summary__question summary__question--{{item.type}}" id="{{id}}">{{item.question | safe}}</dt>
+        <dd class="summary__answer summary__answer--{{item.type}}">
+
+          <div class="summary__answer-text" id="{{id}}-answer">
+
+            {% if item.type == 'date' %}
+
+              {{item.answer[0].value.strftime('%d %b %Y')}} to {{item.answer[1].value.strftime('%d %b %Y')}}
+
+            {% elif item.type == 'checkbox' %}
+
+              {% if item.answer %}
+
+              <ul class="u-m-no">
+                {% for answer in item.answer %}
+                  <li>{{answer}}</li>
+                {% endfor %}
+              </ul>
+
+              {% else %}
+
+                {{not_answered}}
+
+              {% endif %}
+
+            {% elif item.type == 'textarea' %}
+
+              {{item.answer|nl2br or not_answered}}
```

Comment:
I'm getting a string "None" here instead of a falsy value when the comments aren't entered.